src/lexer.coffee

```diff
@@ -919,15 +922,7 @@ exports.Lexer = class Lexer
         ]
       ]
       lparen[2] = lparen.origin[2]
-      rparen = @token 'STRING_END', ')'
-      rparen[2] =
-        first_line:            lastToken[2].last_line
-        first_column:          lastToken[2].last_column
-        last_line:             lastToken[2].last_line
-        last_column:           lastToken[2].last_column
-        last_line_exclusive:   lastToken[2].last_line_exclusive
-        last_column_exclusive: lastToken[2].last_column_exclusive
-        range:                 lastToken[2].range
+      rparen = @token 'STRING_END', ')', offset: endOffset - (quote ? '').length, length: quote?.length ? 0
```

Comment:
By passing `endOffset` into all calls to `mergeInterpolationTokens()`, we're able to accurately encode the location data of the closing quote/delimiter